### PR TITLE
fix twox64concat hasher

### PR DESCRIPTION
**Pull request by haerdib**

### CODE REVIEW

**Comment by haerdib** [comment]
Ups, there is still a bug. Converting it to draft for now

File: src/node_metadata.rs [hunk 1]
```diff
@@ -635,6 +635,12 @@ fn key_hash<K: Encode>(key: &K, hasher: &StorageHasher) -> Vec<u8> {
         StorageHasher::Blake2_256 => sp_core::blake2_256(&encoded_key).to_vec(),
         StorageHasher::Twox128 => sp_core::twox_128(&encoded_key).to_vec(),
         StorageHasher::Twox256 => sp_core::twox_256(&encoded_key).to_vec(),
-        StorageHasher::Twox64Concat => sp_core::twox_64(&encoded_key).to_vec(),
+        StorageHasher::Twox64Concat => {
+            sp_core::twox_64(bytes)
```
Comment (brenzi) [inline src/node_metadata.rs:639]: where does "bytes" come from? compare to a few lines up. shouldn't it work the same way?

Comment (haerdib) [reply src/node_metadata.rs:639]: Yup, that's  the bug - I was a bit a rash with the PR. Fix is incoming

Comment (haerdib) [reply src/node_metadata.rs]: Fixed

**Pull request merged**